src/AnalysisRunner.ts

```diff
@@ -60,7 +62,7 @@ export class AnalysisRunner
 
         // # Scope arguments
 
-        if(options.solution) {
+        if(options.solution && (!options.project && !options.website || options.solution != this.DEFAULT_SOLUTION_VALUE)) {
```

Comment:
I'm not sure about this. If I understand correctly, if user specifies `solution: '.\\*.sln' ` and `project: '*.csproj'`, we will ignore solution file. But if user specifies `solution: '*.sln'` or `solution: 'some.sln'` plus a project, we will not ignore solution and command line will be still invalid.

Maybe we could either report somehow to user that command line is invalid, or ignore options which are invalid (with warning/console log?) but for all invalid values?

Reply:
I am not sure if we should decide for the user which variant of cmd is the desired one. User will already get failed workflow run, and in the details he will get information that the cmd was invalid. Also, he will have information that dotTEST exited with exit code specific to invalid commandline (we report each non-zero exit code, please see run-dottest-analyzer.ts).

Also, other tools does not have any validation. In fact user has to specify manually most of the switches for Jtest and CppTest.

Reply:
It turns out to be problematic - github api will return the default value for the parameter if parameter is not explicitly specified by user. So it's not possible to detect that user has not specified any value for parameters with default value.

So I guess the current solution is good enough.